tests/integration-tests/tests/update/test_update/test_update_instance_list/pcluster.config.update.yaml

```diff
@@ -0,0 +1,22 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+Scheduling:
+  Scheduler: slurm
+  SlurmQueues:
+    - Name: queue1
```

Comment:
Should we include the *AllocationStrategy* here?
Or is this implicitly a test to confirm that the default *AllocationStrategy* is `lowest-price`?

Reply:
This to confirm the default is the expected one.